repos/system_upgrade/el7toel8/actors/prepareupgradetransaction/libraries/preparetransaction.py

```diff
@@ -52,9 +53,19 @@ def permission_guard():
     # FIXME: Not implemented yet. Is it even useful?
     raise NotImplementedError
 
+def _logging_handler(fd_info, buffer):
```

Comment:
Why are you implementing the function? I know, it has "_" underscore, but it's completely same function as in the framework. I would think about the rename (e.g. default_handler) in the framework so there will be something default that could be used. (what do you think guys? @vinzenz, @shaded-enmity, @fernflower)

Reply:
+1 on providing something like a `standard_io_handler` for these purposes

Reply:
can this be done in follow-up PR?
I would like this to be merged ASAP as it is quite difficult to debug errors in transaction